kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -183,6 +184,17 @@ public void put(Collection<SinkRecord> records) throws ConnectException {
       String topic = record.topic();
       int partition = record.kafkaPartition();
       TopicPartition tp = new TopicPartition(topic, partition);
+      if (record.value() == null) {
```

Comment:
I suggest fleshing out in a `maybeSkipOnNullValue` method that returns a `boolean`

Reply:
cc @cyrusv

Reply:
addressed now!